src/vorta/application.py

```diff
@@ -325,7 +325,10 @@ def check_failed_response(self, result: Dict[str, Any]):
             if returncode == 1:
                 # warning
                 msg.setIcon(QMessageBox.Icon.Warning)
-                text = self.tr('Borg exited with a warning message. See logs for details.')
+                text = trans_late(
+                    'messages',
+                    f'Borg exited with a warning message. See <a href="file://{LOG_DIR}">logs</a> for details.',
```

Comment:
For platform independency `pathlib.Path.as_uri()` would find good use here.

Reply:
Should I replace this for the `Log` text in the misc tab in this PR too?

Reply:
That would be great. It doesn't matter for Linux and MacOS support but in anticipation of the Windows alpha being worked on in #1592 it does.

Reply:
Cool, thanks for the context.

Reply:
I just checked `config.py`. 
isn't this enough for platform independence?
```python
dirs = appdirs.AppDirs(APP_NAME, APP_AUTHOR)
LOG_DIR = dirs.user_log_dir
```

Reply:
Windows path use backslashes instead of forward slashes and they have drive names.